completed prelude v1 definition confirmed all sweetests

evomimic opened the pull request.

owleyeview commented:
I noticed that you added "crates/holons_prelude", twice in the root Cargo.toml(lines 26 and 28).  Also wondering if you want to leave all the commented out imports that are replaced by prelude in the tests.  It seems like you might as well remove them.

evomimic commented:
> I noticed that you added "crates/holons_prelude", twice in the root Cargo.toml(lines 26 and 28). Also wondering if you want to leave all the commented out imports that are replaced by prelude in the tests. It seems like you might as well remove them.

Good catch!

evomimic commented:
I decided to make some additional changes to clean up some long-standing issues:

1. Dropped descriptors zome and core_schema zome
2. Renamed `create_empty_tranisent_holon` to simply `new_holon`
3. Dropped use of ` _api `suffix in the holon_operations_api methods -- it was inconsistently applied and redundant
4. Added _internal suffix to HolonServiceApi methods and added doc comment to clarify its usage vs. holon operations api
5. Updated flake.nix to eliminate the CMake errors and "--target arm64-apple-macosx != arm64-apple-darwin" warnings
